src/apps/external_executor/external_executor.cpp

```diff
@@ -264,16 +268,20 @@ namespace externalexecutor
           ccf::endpoints::CommandEndpointContext& ctx,
           externalexecutor::protobuf::IndexStrategy&& payload)
         -> ccf::grpc::GrpcAdapterResponse<google::protobuf::Empty> {
-        auto it = map_index_strategies.find(payload.strategy_name());
-        if (it != map_index_strategies.end())
+        auto executor_id = get_caller_executor_id(ctx);
+        auto it = index_streams.find(executor_id);
+
+        if (it == index_streams.end())
         {
-          it->second->is_indexer_active = false;
+          LOG_INFO_FMT("Strategy NOT FOUND!");
```

Comment:
```suggestion
          LOG_INFO_FMT("No such strategy: {}", payload.strategy_name());
```

Not sure about INFO here, but I think we want to avoid all caps and exclamation marks in log messages.

Reply:
Sorry, that was for my own logging, so that it's easy to search :)

Reply:
I removed this log statement altogether (we return an error anyway)